examples/rondpoint/src/rondpoint.udl

```diff
@@ -2,7 +2,7 @@ namespace rondpoint {
   Dictionnaire copie_dictionnaire(Dictionnaire d);
   Enumeration copie_enumeration(Enumeration e);
   sequence<Enumeration> copie_enumerations(sequence<Enumeration> e);
-  record<DOMString, Enumeration> copie_carte(record<DOMString, Enumeration> c);
+  record<DOMString, EnumerationAvecDonnees> copie_carte(record<DOMString, EnumerationAvecDonnees> c);
```

Comment:
A big +1 for the french :)

(On a completely less serious note, I wonder if the French in rondpoint is a bigger risk than the parser in the shorter term ;)

Reply:
On the other hand, trying to switch this away from French would be like going through the Sync codebase on the removing all the puns...I don't think I'm ready for that just yet 😢 😅

Reply:
How about writing newly added code in English, and changing bits of codes that we touch incrementally, as it makes sense?